Client/Frontend/Login Management/LoginListViewController.swift

```diff
@@ -421,6 +494,10 @@ private class LoginCursorDataSource: NSObject, UITableViewDataSource {
         cell.style = .IconAndBothLabels
         cell.updateCellWithLogin(login)
 
+        let favicon = bestFittingFaviconForLogin(login, toFitImageSize: cell.iconImageView.frame.size)
+        if let faviconURL = favicon?.url.asURL {
+            cell.iconImageView.sd_setImageWithURL(faviconURL, placeholderImage: UIImage(named: "faviconFox"))
```

Comment:
Each call to `sd_setImageWithURL` will construct a new `UIImage(named: "faviconFox")`. I suggest you compute that once.